kubernetes/samples/charts/nginx/samples/tls.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+apiVersion: voyager.appscode.com/v1beta1
+kind: Ingress
+metadata:
+  name: voyager-ssl-1
+  namespace: weblogic-domain1
+  annotations:
+    ingress.appscode.com/type: 'NodePort'
+    ingress.appscode.com/stats: 'true'
+    ingress.appscode.com/affinity: 'cookie'
+spec:
+  tls:
+  - secretName: domain1-tls-cert
+    hosts:
+    - domain1.org
+  rules:
+  - host: domain1.org 
+    http:
+      nodePort: '30305'
+      paths:
+      - backend:
+          serviceName: domain1-cluster-cluster-1
+          servicePort: '8001'
+---
+apiVersion: voyager.appscode.com/v1beta1
+kind: Ingress
+metadata:
+  name: voyager-ssl-2
+  namespace: weblogic-domain2
+  annotations:
+    ingress.appscode.com/type: 'NodePort'
+    ingress.appscode.com/stats: 'true'
+    ingress.appscode.com/affinity: 'cookie'
```

Comment:
change the annotations to use Nginx related.

Reply:
done